starter-code/scripts/articleView.js

```diff
@@ -37,36 +38,53 @@ articleView.handleAuthorFilter = function() {
     //         defining. "$(this)" is using jQuery to select that element, so we can chain jQuery methods
     //         onto it.
     if ($(this).val()) {
-      // TODO: If the select box was changed to an option that has a value, we need to hide all the articles,
+      // Done: If the select box was changed to an option that has a value, we need to hide all the articles,
       //       and then show just the ones that match for the author that was selected.
       //       Use an "attribute selector" to find those articles, and fade them in for the reader.
+      $('article').filter((ind, art) => $(art).attr('data-author') != $(this).val()).each(
+        (ind, art) =>$(art).hide());
 
+      var selectedItem = $('article').filter((ind, art) => $(art).attr('data-author') === $(this).val())[0];
+      $(selectedItem).show();
     } else {
-      // TODO: If the select box was changed to an option that is blank, we should
+      // Done: If the select box was changed to an option that is blank, we should
       //       show all the articles, except the one article we are using as a template.
-
+      $('article').each((ind, art) => $(art).show());
```

Comment:
This is unnecessarily complicated. Since jQuery automatically applies each method call to all the elements, you only need `$('article').show()`.